### ISSUE TITLE
No detection - Samsung Galaxy S7 - 1024x768

### ISSUE THREAD

**Issue by Arcorme** [artoolkit/arunity5#123]
Hello,

I use ARToolkit v5.3.2 for Unity and publish an Android app for a Galaxy S7.
I managed to change camera resolution and tested several resolutions by changing "Plugins/Android/ResLib-release.aar" values.xml file and rezipping it.
This works well.

For resolutions 640x480 or 1280x720 for example the calibration is not perfect but the detection works.
(The application starts, downloads calibration information for several resolutions and stores it in cache folder under Android application folder in "cparam_cache.db" file)

My trouble is that using 1024x768 resolution there is a strange thing happening : 
IF there is no internet then the detection works in default mode (pretty well)
IF there is internet then "cparam_cache.db" is created and contains 1024x768 resolution but in the application there is no detection at all ...

The steps :
1) I set resolution to 1024x768 and check in the log that it is the effective resolution.
2) I upload application to phone, check internet connectivity and launch it ... no detection ...
3) I open the file "cparam_cache.db" created in the cache folder of the application and check that there is the 1024x768 resolution and data in the camera_para_base64 column.
4) I don't know why it doesn't work ?

What is the solution to have it working with 1024x768 resolution please ?

Thank you,

Arcorme

**Comment by ThorstenBux** [comment]
Did you do a calibration for your device with a 1024x768 resolution?

**Comment by Arcorme** [comment]
Yes, using the application "ARToolKit Camera Calibration" on Android.
I did it several times to be sure the calibration worked.

**Comment by ThorstenBux** [comment]
Can your tell me your exact device ID. I might be able to have a look at the calibration database and see if the calibration is good or bad. Unfortunately, Samsung uses different IDs for the S7 series:
https://www.techwalls.com/samsung-galaxy-s7-edge-model-numbers-differences/

Would be helpful if you could provide the detailed ID.

Thanks

**Comment by Arcorme** [comment]
The phone I use is a Samsung Galaxy S7 (SM-G930F)

**Comment by OptrixAU** [comment]
I mentioned in a previous issue that I had problems myself at some resolutions, but the results were perfect at Full-HD.

These problems were also showing up in the ARToolkit demos, which indicates that they were problems with the base toolkit, not the Unity version.
